config/samples/core_v1beta1_openstackcontrolplane.yaml

```diff
@@ -114,6 +114,10 @@ spec:
       databaseInstance: openstack
       containerImage: quay.io/tripleozedcentos9/openstack-neutron-server:current-tripleo
       secret: osp-secret
+  horizon:
```

Comment:
please also update the network isolation samples

Reply:
ack. Done

Reply:
@bshephar not sure I see the network-isolation related samples updated in this patch